build/cockroach.rb

```diff
@@ -30,11 +30,50 @@ def install
     system "xcrun", "make", "GOFLAGS=-v", "-C",
            "src/github.com/cockroachdb/cockroach", "build"
     bin.install "src/github.com/cockroachdb/cockroach/cockroach" => "cockroach"
+
+    # TODO(pmattis): Debug the launchctl stuff
+    # (prefix+'com.cockroachlabs.cockroachdb.plist').write startup_plist
+    # (prefix+'com.cockroachlabs.cockroachdb.plist').chmod 0644
   end
 
   test do
     system "#{bin}/cockroach", "version"
   end
 
-  # TODO(peter): Need to figure out how set cockroach to running.
+  def caveats
+    <<-EOS.undent
+    Start cockroach:
+        cockroach start --store=/usr/local/cockroach-data
```

Comment:
the postgres formula uses `#{HOMEBREW_PREFIX}/var/postgres` which is identical to `#{var}/postgres`, which translates to `#{var}/cockroach`, which is what I originally suggested.